currency/forexprovider/forexprovider.go

```diff
@@ -3,18 +3,19 @@
 package forexprovider
 
 import (
+	"errors"
+
 	"github.com/thrasher-/gocryptotrader/currency/forexprovider/base"
 	currencyconverter "github.com/thrasher-/gocryptotrader/currency/forexprovider/currencyconverterapi"
 	"github.com/thrasher-/gocryptotrader/currency/forexprovider/currencylayer"
 	exchangerates "github.com/thrasher-/gocryptotrader/currency/forexprovider/exchangeratesapi.io"
 	fixer "github.com/thrasher-/gocryptotrader/currency/forexprovider/fixer.io"
 	"github.com/thrasher-/gocryptotrader/currency/forexprovider/openexchangerates"
-	log "github.com/thrasher-/gocryptotrader/logger"
 )
 
-// ForexProviders is an array of foreign exchange interfaces
+// ForexProviders is a foreign exchange handler type
 type ForexProviders struct {
-	base.IFXProviders
+	base.FXHandler
 }
 
 // GetAvailableForexProviders returns a list of supported forex providers
```

Comment:
Should this be hardcoded here? How to add a new forexProvider?

Reply:
This is being used by our configuration system, just so as we add we populate the config.json appropriately. So for ease you can just add your forex provider name here. If you have a better way of doing things let me know.